agimus_controller/agimus_controller/mpc_output.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class MPCOutputData:
+    """Output data structure of the MPC."""
+
+    x: list  # List of numpy.ndarray, the output state.
+    u: list  # List of numpy.ndarray, the output control.
+    ricatti_gains: list  # List of numpy.ndarray, the Ricatti gains.
+    feed_forward_terms: list  # List of numpy.ndarray, the feed forward terms.
```

Comment:
There is needed a util that will do conversion to `lfc_py_types.Control` so there is no need to do the tedious job of type conversion

Reply:
I am not sure I understand your comment here.